queue_processors/queue_processor/README.md

```diff
@@ -1,10 +1,7 @@
-## QueueProcessor
+## queue_processor module
 
-These scripts handle the messages added to the following queues:
-* `/DataReady`
-* `/ReductionComplete`
-* `/ReductionError`
+This modules handles the majority of message management in the autoreduction system. It is responsible for the intial creation of the database records for reduction jobs as well as updating the status of reduction records as the message traverses through the system. 
 
-As well as this, the Queue processor is the interface between the message queues and
-the autoreduction database. More specifically it creates the dataobject for each run,
-and updates it's status as it progresses through pipeline of the system.
\ No newline at end of file
+The `queue_listener.py` is responsible for ingesting and distributing messages to their correct functions for processing. The `queue_listner` consumes messages from the `/DataReady`, `/ReductionStarted`,`/ReductionComplete`, `/ReductionError` and `/ReductionSkipped` queues. The service is run as a daemon service and is such only runs on linux. The daemon service is controlled by `queue_processor_daemon.py` which give a command line interface to start/stop/restart the daemon process. However, the daemon is normally controlled in tandem with the `autoreduction_processor` via the `../restart.sh` script that restarts both these processes. 
```

Comment:
Suggestions:
* The service is run as a daemon process - as such only runs on Linux. 
* Slight changes to 4th sentence.
```suggestion
The `queue_listener.py` is responsible for ingesting and distributing messages to their correct functions for processing. The `queue_listner` consumes messages from the `/DataReady`, `/ReductionStarted`,`/ReductionComplete`, `/ReductionError` and `/ReductionSkipped` queues. The service is run as a daemon process - as such only runs on Linux.  The daemon process is controlled by `queue_processor_daemon.py`, which produces a command line interface to start/stop/restart the daemon process. 
However, the daemon is normally controlled in tandem with the `autoreduction_processor` via the `../restart.sh` script that restarts both these processes. 
```